src/components/card/_card.scss

```diff
@@ -0,0 +1,92 @@
+$cardBorderColor: $graySecondaryLight;
+$cardBorderSize: 1px;
+$cardShadow: 0 1px 3px 2px rgba(0, 0, 0, 0.05);
+
+$includeHtml: false !default;
+
+@if ($includeHtml) {
+
+  .sg-card {
+    @include component();
+    display: inline-flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    border-radius: $borderRadiusDefault;
+    border: $cardBorderColor solid $cardBorderSize;
+    overflow: hidden;
```

Comment:
I think we could do it on holes - would make its more flexible?

Reply:
@aju this is necessary here, as holes have background color, and parent have border-radius?